releasenotes/notes/update-sentence-transformers-3-3bca4e3f3ad3e7ba.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Starting from `haystack-ai==2.4.0`, Haystack is compatible with `sentence-transformers>=3.0.0`.
```

Comment:
maybe change "compatible" with "requires" or does Haystack still works with older versions of sentence-transformers?

Reply:
Thanks. I have made the note clearer.
https://github.com/deepset-ai/haystack/pull/8295/commits/36f92e58ed21c22e2fffe9af0ed2a174f213d2a5